docker/1.4.4/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM alpine:3.3
+MAINTAINER David Audet <david.audet@ca.com>
+
+LABEL Description="Mosquitto MQTT Broker"
```

Comment:
@karlp "Mosquitto" is a trademark of the Eclipse Foundation, and the trademark guidelines dictate that the first mention of the project must use its formal name prefixed with Eclipse.
You can read more at https://www.eclipse.org/projects/handbook/index.php#trademarks